extensions/positron-python/src/client/positron/session.ts

```diff
@@ -107,7 +107,10 @@ export class PythonRuntimeSession implements positron.LanguageRuntimeSession, vs
 
         const interpreter = interpreterService.getInterpreters().find((i) => i.id === extraData.pythonEnvironmentId);
         if (!interpreter) {
-            throw new Error(`Interpreter not found: ${extraData.pythonEnvironmentId}`);
+            const interpreterIds = interpreterService.getInterpreters().map((i) => `${i.id}`);
```

Comment:
For readability, maybe we could do something like this?

```suggestion
            const interpreterIds = interpreterService.getInterpreters().map((i) => `\n- ${i.id}`);
```

I think this should result in output that looks like:

```
ERR Interpreter /Users/juliasilge/.pyenv/versions/3.11.7/envs/polars-testing/bin/python not found in available Python interpreters:
- /Users/juliasilge/.pyenv/versions/3.10.12/bin/python,
- /Users/juliasilge/.pyenv/versions/3.10.13/bin/python,
- /Users/juliasilge/.pyenv/versions/3.10.9/bin/python,
- /Users/juliasilge/.pyenv/versions/3.11.5/bin/python,
- /Users/juliasilge/.pyenv/versions/3.11.6/bin/python,
- /Users/juliasilge/.pyenv/versions/3.11.7/bin/python,
- ...
```